src/MAUI/Maui.Samples/Samples/Map/AuthorMap/AuthorMap.xaml.cs

```diff
@@ -33,13 +27,13 @@ public partial class AuthorMap : ContentPage
         private const string ArcGISOnlineUrl = "https://www.arcgis.com/sharing/rest";
 
         // String array to store names of the available basemaps
```

Comment:
Many comments had periods added to the end of them, but this comment and the one on line 39 did not. Is this intentional (perhaps to differentiate between preliminary declarations and instructions)? This seems inconsistent across samples. For example, Browse Building Floors has periods at the end of every comment, but Display Map has none. Might be worth considering a separate PR for establishing consistency across comments, would also make such changes more relevant.